### DISCUSSION TITLE
Keyball46 ジャンパのランド剥がれ

### DISCUSSION THREAD

**Discussion by M1lkTea114** [Yowkees/keyball#168] [Q&A] [answered]
お世話になっております。
当方、今年の1月ごろにkeyball46を購入させていただき、先日まで問題なく利用できていたのですが
pro microのusbコネクタがもげてしまい、新しいpro microと交換したところトラックボールとNキーが認識されなくなりました。
（右手トラックボールで利用しています）
背面のはんだに問題があるのかと考え、メンテナンスがてら調整していたのですがその際に、添付画像のball PCB 直上のジャンパ部分のはんだがはがれてしまい、同時にランドまで取れてしまいました（左から2つとも）。
こちらを導線などを用いて修復を行いたいのですが、どことどこを繋げばいいのか見当がつきません。
類似のランド剥げのQ&Aも確認しましたがいずれもトラックボールがない側のOLED箇所だったため解決できませんでした。
お忙しいところ申し訳ありませんが、解決策等ありましたら教えていただけますでしょうか。
![P_20221220_231720](https://user-images.githubusercontent.com/79870599/208691439-566298e8-9b2f-471e-b94c-0ab34466fe4c.jpg)

**Comment by rinyacho** [comment]
正確なことは分からないのですが、回路図を見る限りballpcbとpromicroの該当箇所をショートさせれば動くと思います。
SCLK＆NCSか、MISO＆MOSIか、どちらかのセットだとは思うのですが、情報足りてなくて正確なところは分かりかねます。
導通確認をしながら、回路図を追っていくと分かると思います。
回路図の抜粋載せておきますね。ご参考までに〜。

![スクリーンショット 2023-01-04 21 43 47](https://user-images.githubusercontent.com/97168861/210558755-709ac852-3aa1-46d9-9dca-10205be0c5ad.png)

**Comment by Yowkees** [comment, marked as answer]
ご返信が大変遅くなり申し訳ありませんでした。

下記画像の同色の端子をエナメル線や被覆銅線で接続していただければOLEDを復活できます。
お手数ですがご対応よろしくお願いいたします。

![46Right_fix](https://user-images.githubusercontent.com/84209772/211182536-539fddf2-1a64-4b66-b594-fea4b7764dcf.jpg)

rinyacho様もご回答ありがとうございました。

**Comment by M1lkTea114** [comment]
@Yowkees 様、 @rinyacho 様。
いただいた内容で接続したところ正常に動作させることができました。
ご回答ありがとうございました。
